test/algorithms/covered_by/test_covered_by.hpp

```diff
@@ -76,7 +76,7 @@ void test_geometry(std::string const& wkt1,
     boost::variant<Geometry1> v1(geometry1);
     boost::variant<Geometry2> v2(geometry2);
 
-    typedef typename bg::strategy::covered_by::services::default_strategy
+    typedef typename bg::strategies::relate::services::default_strategy
```

Comment:
since you changed it maybe use `using` ?

Reply:
Done